src/gdsync.cpp

```diff
@@ -452,15 +459,44 @@ static int gds_fill_poll(gds_op_list_t &ops, CUdeviceptr ptr, uint32_t magic, in
                 retcode = EINVAL;
                 goto out;
         }
+#if 0        
```

Comment:
this code should query the GPU for CU_DEVICE_ATTRIBUTE_CAN_FLUSH_REMOTE_WRITES and use the flusher on if required

Reply:
the required infrastructure code is being developed in a separate branch